<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">QFrame{
border-radius:10px;
background-image: url(image/BackGraund.jpg);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
background: rgba(0,0,0,0);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>548</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="Bt_min">
            <property name="styleSheet">
             <string notr="true">QPushButton{
border-radius:18px;
}
QPushButton:hover{
background-color: rgb(0,0,0);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>src/Imagenes_UI/Barra-removebg-preview.png</normaloff>src/Imagenes_UI/Barra-removebg-preview.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>70</width>
              <height>70</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Bt_normal">
            <property name="styleSheet">
             <string notr="true">QPushButton{
border-radius:18px;
}
QPushButton:hover{
background-color: rgb(0,0,0);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>src/Imagenes_UI/Cuadrado2-removebg-preview.png</normaloff>src/Imagenes_UI/Cuadrado2-removebg-preview.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>50</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Bt_max">
            <property name="styleSheet">
             <string notr="true">QPushButton{
border-radius:18px;
}
QPushButton:hover{
background-color: rgb(0,0,0);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>src/Imagenes_UI/Cuadrado-removebg-preview.png</normaloff>src/Imagenes_UI/Cuadrado-removebg-preview.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>60</width>
              <height>80</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Bt_close">
            <property name="styleSheet">
             <string notr="true">QPushButton{
border-radius:18px;
}
QPushButton:hover{
background-color: rgb(0,0,0);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>src/Imagenes_UI/x-removebg-preview.png</normaloff>src/Imagenes_UI/x-removebg-preview.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_superior">
         <property name="styleSheet">
          <string notr="true">QFrame{
background: rgba(0,0,0,0);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>274</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSplitter" name="splitter">
            <property name="styleSheet">
             <string notr="true">QFrame{
background: rgba(0,0,0,0);
}</string>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QFrame" name="frame_4">
             <property name="maximumSize">
              <size>
               <width>600</width>
               <height>800</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QFrame{
background: rgba(0,0,0,0.3);
}


QPushButton#pushButton{

background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(20, 47, 78, 219), stop:1 rgba(85, 98, 112, 226));

color:rgba(255, 255, 255, 210);
 border-radius:5px; 
}

QPushButton#pushButton:hover{

background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(40, 67, 98, 219), stop:1 rgba(105, 118, 132, 226));
 }

QPushButton#pushButton:pressed{

padding-left:5px;

padding-top:5px;

background-color:rgba(105, 118, 132, 200);
}</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label">
                <property name="font">
                 <font>
                  <family>Kristen ITC</family>
                  <pointsize>15</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(250, 250, 250);</string>
                </property>
                <property name="text">
                 <string>Iniciar Sesion</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="User">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgba(0, 0,  0, 0);
border: none;
border-bottom: 2px solid rgba(105, 118, 132, 255);
color: rgba(255, 255, 255, 230);</string>
                </property>
                <property name="placeholderText">
                 <string>UserName</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="Password">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgba(0, 0,  0, 0);
border: none;
border-bottom: 2px solid rgba(105, 118, 132, 255);
color: rgba(255, 255, 255, 230);</string>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
                <property name="placeholderText">
                 <string>Password</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox">
                <property name="font">
                 <font>
                  <family>Kristen ITC</family>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color:rgb(255,255,255);</string>
                </property>
                <property name="text">
                 <string>Recordar contraseña</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="Enter">
                <property name="font">
                 <font>
                  <family>Kristen ITC</family>
                  <pointsize>15</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{

background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(20, 47, 78, 219), stop:1 rgba(85, 98, 112, 226));

color:rgba(255, 255, 255, 210);
 border-radius:5px; 
}

QPushButton:hover{

background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(40, 67, 98, 219), stop:1 rgba(105, 118, 132, 226));
 }

QPushButton:pressed{

padding-left:5px;

padding-top:5px;

background-color:rgba(105, 118, 132, 200);
}</string>
                </property>
                <property name="text">
                 <string>Enter</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="font">
                   <font>
                    <family>Kristen ITC</family>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(250, 250, 250);</string>
                  </property>
                  <property name="text">
                   <string>¿No tienes cuenta?</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="Reg">
                  <property name="font">
                   <font>
                    <family>Kristen ITC</family>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton{
background:rgba(0,0,0,0);
}
QPushButton:hover{
border-radius:10px;
color: rgb(52, 143, 255);
border: None;
background-color: None;
}</string>
                  </property>
                  <property name="text">
                   <string>Registrate</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>273</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="src/Imagenes_UI/back.qrc"/>
 </resources>
 <connections/>
</ui>
